step5/index.html

```diff
@@ -65,18 +65,26 @@
     this.root.innerHTML = `
 <style>
 :host {
-  background: #eee;
+  background: #fff;
```

Comment:
you just changed this in the last step?
(For better or worse, the grey background makes it easier to see where to drag and drop)